### DISCUSSION TITLE
"JWST-JADES. Possible Population III signatures at z=10.6 in the halo of GN-z11"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#26] [Astrophysics and Physics]
The "first" stars in the universe?
[arXiv:2306.00953](https://arxiv.org/abs/2306.00953)

Certainly not!

**Comment by khuramonline** [comment]
They can "conclude" POPIII stars but they cannot "see" even farther away galaxies and clusters:

[100% visible bright Sky shown by JWST](https://www.quora.com/How-can-scientists-say-that-Galaxy-Glass-z13-is-one-of-the-farthest-visible-Galaxy-when-countless-other-galaxies-are-visible-in-the-JWST-image-right-in-the-background-of-Glass-z-13/answer/Khuram-Rafique)

**Reply by RedshiftDrift** [reply]
GN-z11 was given a spectroscopic confirmation of the redshift.

It is false to claim that the distance measurement of GN-z11 is only based on the colour of a "red dot".  The argument you make on Quora is incorrect as it ignores the spectroscopic analysis of the data that was made to reach the conclusions.

**Reply by khuramonline** [reply]
Let me explain Sir.

I have absolutely not written anywhere that distance of GNz-11 or any other galaxy was determined based on color of red dot.

I also have never denied the application of spectroscopic analysis.

I have never stated any such thing. I request you to recheck 

I may have lots of other claims. But these are not my claims..

**Reply by RedshiftDrift** [reply]
[You write](https://www.quora.com/How-can-scientists-say-that-Galaxy-Glass-z13-is-one-of-the-farthest-visible-Galaxy-when-countless-other-galaxies-are-visible-in-the-JWST-image-right-in-the-background-of-Glass-z-13/answer/Khuram-Rafique): "So I say that all the red dots in the background of the main image are galaxies."
These are likely noise on the image, and you have no spectroscopic data to suport that claim.

Then you add "So far I have stated it without giving evidence."
This group has no interest in claims that are not supported by evidence.

**Reply by khuramonline** [reply]
Yes Sir, this is my claim. These are not only galaxies. These also include many clusters of galaxies. 

"These are noise" --- this is also a claim. 

Just because spectroscopic data is not available does not necessarily mean that they are not galaxies or clusters.

Yes following are my words:

"So far I have stated it without giving evidence."

After writing this following was my next sentence:

"So evidence also will come (i.e. within rest of my answer)".

Here I share only one evidence. Other evidences are also available in my quora answer. What evidence I share here is also included in my answer:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/f53f0176-b2c8-452f-a405-270ce663f14a)

Above is a screenshot from a NASA information video. This is pre-JWST image -- It is analysis/ tour video of galaxies cluster Abell 2744.

With above image that is about an HST image, it is clear that NASA had acknowledged the existence of these dots as galaxies in the pre-JWST days.

According to above image, these red dots are normally not visible in the HST images and become visible only when magnified and brightened through the gravitational lensing of intervening large clusters.

The pre-JWST stance of NASA about them was that these are distant galaxies that exist at distance 13+ billion light years. **What JWST has shown now is that these dots exist even in the background of those galaxies which are officially located at distance of 13.5 billion light years.**

**This image also makes it clear that red-dots can be enlarged due to the application of gravitational lensing.**

**Off course these are not properties of noise.** 

Sir ... I have radically different claims. And my claims are not without supporting reasons or evidence.

I invite you to keep looking at the image of glass-z13 ... then again read my quora answer and then again see the same image of galaxy glass-z13

and off course if you think that my claim is without any supporting reason or evidence or it is just a baseless claim ... you can remove my membership on this forum which will clearly tell me that I have not to further participate here.

**Reply by khuramonline** [reply]
Following is earlier form of my idea about red dots. By August 04, 2022, I was aware of only background **red dots** (i.e. by that time I was not aware about green and blue dots).

[Evidence that background reddish dots in the background of the image of Galaxy Glass-z13 are Extremely redshifted far off Galaxies (August 04, 2022)](https://zenodo.org/record/6962957)

Within that earlier perspective, I already had addressed to objections that I now face here.

> **The researchers who discovered the galaxy Glass-z13  have  not  denied  the  existence  of  small  reddish  dots**  as they  have just  discussed “two particularly luminous sources” which they present as galaxies Glass-z13 and Glass-z11 respectively  in  their  paper. **Seems  like  luminous  sources  below  certain  level  were  of  no interest to them** and they did not discuss them in detail. Anyhow, if small reddish dots in the background of galaxy Glass-z13 are mere distortion of the image and not real in the sense of massive objects then it is another question on the quality of the images of JWST. Fortunately we have indications that these smaller red dots are in fact galaxies. Well, simple red dots, if these  are  merely  visible  dots  and  nothing  else,  **cannot  be  magnified  through  the  gravity lensing of the foreground galaxies.**

After that earlier perspective, I had realized that green dots in the background were also galaxies. On same day, I also figured out that blue dots were also galaxies. Thus I duly concluded that sky is 100% bright with visible (to JWST) infrared galaxies and clusters. My (rough) estimate is that sky is 100% filled with such visible objects up to the distance of 1 trillion light years. 

**I started writing on quora that sky is 100% bright in infrared light since September 11, 2022.**
